scripts/build_helpers/bin/build_dart.dart

```diff
@@ -47,7 +47,10 @@ void main(List<String> args) async {
       exit(-1);
     }
 
-    if (!await _buildDart()) {
+    if (!await _buildDart('release')) {
+      exit(-1);
+    }
+    if (!await _buildDart('debug')) {
```

Comment:
Since this script is used as part of the github action / CI, it may be worth adding an argument instead of always building both versions.